app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -75,6 +76,12 @@ def updated_at
       def publish_messages
         Messages.publish(@model.sequencing_runs, Pipelines.pacbio.message)
       end
+
+      def volume_tracking_attributes
+        return unless Flipper.enabled?(:y24_153__expose_volume_tracking_attributes_on_pool_resource)
+
+        attributes :used_volume, :available_volume
```

Comment:
Testing locally, I dont think this is working. Seems to error when I try and access the pools endpoint with the flag on. I think we should be fine to remove this use of the feature flag entirely and just keep used_volume and available_volume exposed as the UI can now handle it.